### ISSUE TITLE
virtualization detection code prints /proc/cpuinfo contents in debug mode

### ISSUE THREAD

**Issue by keszybz** [systemd/systemd#3155]
Seems to be too much. In particular, this is printed for every core.

```
Apr 29 21:07:28 laptop systemd-networkd[7705]: No virtualization found in DMI
Apr 29 21:07:28 laptop systemd-networkd[7705]: No virtualization found in CPUID
Apr 29 21:07:28 laptop systemd-networkd[7705]: Virtualization XEN not found, /proc/xen/capabilities does not exist
Apr 29 21:07:28 laptop systemd-networkd[7705]: This platform does not support /proc/device-tree
Apr 29 21:07:28 laptop systemd-networkd[7705]: No virtualization found in /proc/cpuinfo (processor        : 0
Apr 29 21:07:28 laptop systemd-networkd[7705]: vendor_id        : GenuineIntel
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpu family        : 6
Apr 29 21:07:28 laptop systemd-networkd[7705]: model                : 23
Apr 29 21:07:28 laptop systemd-networkd[7705]: model name        : Intel(R) Core(TM)2 Duo CPU     P8700  @ 2.53GHz
Apr 29 21:07:28 laptop systemd-networkd[7705]: stepping        : 10
Apr 29 21:07:28 laptop systemd-networkd[7705]: microcode        : 0xa0b
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpu MHz                : 2533.000
Apr 29 21:07:28 laptop systemd-networkd[7705]: cache size        : 3072 KB
Apr 29 21:07:28 laptop systemd-networkd[7705]: physical id        : 0
Apr 29 21:07:28 laptop systemd-networkd[7705]: siblings        : 2
Apr 29 21:07:28 laptop systemd-networkd[7705]: core id                : 0
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpu cores        : 2
Apr 29 21:07:28 laptop systemd-networkd[7705]: apicid                : 0
Apr 29 21:07:28 laptop systemd-networkd[7705]: initial apicid        : 0
Apr 29 21:07:28 laptop systemd-networkd[7705]: fpu                : yes
Apr 29 21:07:28 laptop systemd-networkd[7705]: fpu_exception        : yes
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpuid level        : 13
Apr 29 21:07:28 laptop systemd-networkd[7705]: wp                : yes
Apr 29 21:07:28 laptop systemd-networkd[7705]: flags                : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx lm constant_tsc arch_perfmon pebs bts nopl aperfmperf pni dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm sse4_1 xsave lahf_lm tpr_shadow vnmi flexpriority dtherm ida
Apr 29 21:07:28 laptop systemd-networkd[7705]: bugs                :
Apr 29 21:07:28 laptop systemd-networkd[7705]: bogomips        : 5053.92
Apr 29 21:07:28 laptop systemd-networkd[7705]: clflush size        : 64
Apr 29 21:07:28 laptop systemd-networkd[7705]: cache_alignment        : 64
Apr 29 21:07:28 laptop systemd-networkd[7705]: address sizes        : 36 bits physical, 48 bits virtual
Apr 29 21:07:28 laptop systemd-networkd[7705]: power management:
Apr 29 21:07:28 laptop systemd-networkd[7705]: processor        : 1
Apr 29 21:07:28 laptop systemd-networkd[7705]: vendor_id        : GenuineIntel
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpu family        : 6
Apr 29 21:07:28 laptop systemd-networkd[7705]: model                : 23
Apr 29 21:07:28 laptop systemd-networkd[7705]: model name        : Intel(R) Core(TM)2 Duo CPU     P8700  @ 2.53GHz
Apr 29 21:07:28 laptop systemd-networkd[7705]: stepping        : 10
Apr 29 21:07:28 laptop systemd-networkd[7705]: microcode        : 0xa0b
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpu MHz                : 2534.000
Apr 29 21:07:28 laptop systemd-networkd[7705]: cache size        : 3072 KB
Apr 29 21:07:28 laptop systemd-networkd[7705]: physical id        : 0
Apr 29 21:07:28 laptop systemd-networkd[7705]: siblings        : 2
Apr 29 21:07:28 laptop systemd-networkd[7705]: core id                : 1
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpu cores        : 2
Apr 29 21:07:28 laptop systemd-networkd[7705]: apicid                : 1
Apr 29 21:07:28 laptop systemd-networkd[7705]: initial apicid        : 1
Apr 29 21:07:28 laptop systemd-networkd[7705]: fpu                : yes
Apr 29 21:07:28 laptop systemd-networkd[7705]: fpu_exception        : yes
Apr 29 21:07:28 laptop systemd-networkd[7705]: cpuid level        : 13
Apr 29 21:07:28 laptop systemd-networkd[7705]: wp                : yes
Apr 29 21:07:28 laptop systemd-networkd[7705]: flags                : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx lm constant_tsc arch_perfmon pebs bts nopl aperfmperf pni dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm sse4_1 xsave lahf_lm tpr_shadow vnmi flexpriority dtherm ida
Apr 29 21:07:28 laptop systemd-networkd[7705]: bugs                :
Apr 29 21:07:28 laptop systemd-networkd[7705]: bogomips        : 5053.92
Apr 29 21:07:28 laptop systemd-networkd[7705]: clflush size        : 64
Apr 29 21:07:28 laptop systemd-networkd[7705]: cache_alignment        : 64
Apr 29 21:07:28 laptop systemd-networkd[7705]: address sizes        : 36 bits physical, 48 bits virtual
Apr 29 21:07:28 laptop systemd-networkd[7705]: power management:
Apr 29 21:07:28 laptop systemd-networkd[7705]: )
Apr 29 21:07:28 laptop systemd-networkd[7705]: This platform does not support /proc/sysinfo
Apr 29 21:07:28 laptop systemd-networkd[7705]: Found VM virtualization none
```

**Comment by poettering** [comment]
yeah, indeed, we shouldn't dump the contents of /proc/cpuinfo, that is really too much...

**Comment by poettering** [comment]
Fix waiting in #3170.
